### PR TITLE
Fix error assertions and cleanup dependencies

### PR DESCRIPTION
**Pull request by ykim-1**
## 📝 Description

replaced all try and except blocks with pytest.raises for cleaner and proper assertion
since mock is part of python std lib, we can import it directly from unittest

## ✔️ How to Test

**What are the steps to reproduce the issue or verify the changes?**

**How do I run the relevant unit/integration tests?**

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

File: tests/unit/test_configuration.py [hunk 1]
```diff
@@ -110,12 +108,11 @@ def test_print_users(self):
 
         f = io.StringIO()
 
-        try:
+        with pytest.raises(SystemExit):
             with contextlib.redirect_stdout(f):
                 conf.print_users()
```
Comment (zliang-akamai) [inline tests/unit/test_configuration.py:113]:
```suggestion
        with pytest.raises(SystemExit) as excinfo:
            with contextlib.redirect_stdout(f):
                conf.print_users()
        assert excinfo.value.code == 0
```

Can you try something like this to assert the error code of the `SystemExit` instance?

The pytest API can allow you to access the instance of the exception, you can see [details docs here](https://docs.pytest.org/en/latest/how-to/assert.html#assertions-about-expected-exceptions).

Comment (ykim-1) [reply tests/unit/test_configuration.py]: Done. Don't know why I added error code assertion in other places but not this file lol..

**Review by zliang-akamai — APPROVED** [review]

**Comment by zliang-akamai** [comment]
Nice work!

**Review by yec-akamai — APPROVED** [review]
lgtm

**Pull request merged**